WebDriverAgentTests/IntegrationTests/XCElementSnapshotHelperTests.m

```diff
@@ -63,35 +63,35 @@ - (void)testParentMatchingType
   XCTAssertEqual(windowSnapshot.elementType, XCUIElementTypeWindow);
 }
 
-- (void)testParentMatchingOneOfTypes
+- (void)testFindVisibleParentMatchingOneOfTypes
 {
   [self goToAttributesPage];
   XCUIElement *todayPickerWheel = self.testedApplication.pickerWheels[@"Today"];
   XCTAssertTrue(todayPickerWheel.exists);
   [todayPickerWheel resolve];
-  XCElementSnapshot *datePicker = [todayPickerWheel.lastSnapshot fb_parentMatchingOneOfTypes:@[@(XCUIElementTypeDatePicker), @(XCUIElementTypeWindow)]];
```

Comment:
And I do have scenario where current PR is helpful. But we can't repro that in integration app